### PR TITLE
25 / FEATURE: Create sale section

### PR DESCRIPTION
**Pull request by iMdPd**
I created a fully styled sale section right above the new furniture section. All boxes have a constant height. I hope I correctly understood the part related to getting data from application state.... I've created new selector to get those products which old price is bigger than actual price. To improve UX, every time you refresh the page list of sale product will be shuffled. Products are selected from application state.  Shuffling products results in a different order of products in the array each time page is refreshed. From now Im using first three product in sale section. I used ternary operator to show percentage decrease of product price. 

Link to Jira task:
https://projects.kodilla.com/browse/WDP230402-25

### CODE REVIEW

File: src/components/features/SaleBoxes/SaleBoxes.module.scss [hunk 1]
```diff
@@ -0,0 +1,142 @@
+@import "../../../styles/settings.scss";
+
+.root {
+  margin-bottom: 2rem;
+
+  .mainBox {
+    height: 500px;
```
Comment (ElizaHynek) [inline src/components/features/SaleBoxes/SaleBoxes.module.scss:7]: zmniejszyłabym do 400px - będzie lepiej wizualnie wyglądało

File: src/components/features/SaleBoxes/SaleBoxes.module.scss [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+        top: 0;
+        left: 0;
+        right: 0;
+        text-align-last: center;
+
+        .collection {
+          font-weight: 300;
+          font-size: 2.5rem;
+          color: $header-topbar-bg;
+          margin: 0;
```
Comment (ElizaHynek) [inline src/components/features/SaleBoxes/SaleBoxes.module.scss:56]: tutaj dodałabym większy margin od góry i pomiędzy collection i price - będzie lepiej wygładało

**Review by ElizaHynek — CHANGES_REQUESTED** [review]

**Comment by iMdPd** [comment]
I've changed SaleBox styles as Eliza asked. I set the styles according to the guidelines of the settings file.

**Review by Evil-Cloud89 — APPROVED** [review]

**Comment by walldy-0** [comment]
Na tablecie kolumna nie jest wyrównana.

image

**Review by walldy-0 — COMMENTED** [review]
Na tablecie kolumna jest niewyrównana.

image

**Comment by iMdPd** [comment]
Dzięki, zmiany zostały wprowadzone.

**Review by walldy-0 — APPROVED** [review]

**Pull request merged**